app/src/main/java/com/imdbmovieapp/data/remote/dto/detailMovieDto/MovieDetailDto.kt

```diff
@@ -0,0 +1,13 @@
+package com.imdbmovieapp.data.remote.dto.detailMovieDto
+
+import com.imdbmovieapp.data.remote.dto.genreMovieDto.Genre
+
+data class MovieDetailDto(
+    val genres: List<Genre>,
+    val id: Int,
+    val overview: String,
+    val poster_path: String,
+    val release_date: String,
```

Comment:
Use `@SerializedName` for Gson and `@Json(name="poster_path")` for Moshi annotations for underscore parameter names and use camelCase approach in every Kotlin file.